src/Microsoft.Management.Deployment.CsWinRTProjection/Microsoft.Management.Deployment.CsWinRTProjection.csproj

```diff
@@ -10,6 +10,10 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IsAotCompatible>true</IsAotCompatible>
     <DisableRuntimeMarshalling>true</DisableRuntimeMarshalling>
+    <!--
+    !!! Remove or update this on the next Microsoft.Windows.CsWinRT package version update. !!!
+    -->
+    <WindowsSdkPackageVersion>10.0.22000.54</WindowsSdkPackageVersion>

```

Comment:
```suggestion
    <WindowsSdkPackageVersion>10.0.22000.53</WindowsSdkPackageVersion>

```

Reply:
.53 also supports .NET 8. The UWP support is not exclusive to .54 (the phrasing in our release notes isn't entirely accurate). The .54 version just also has the projections for `Windows.UI.Xaml.*` types, but we're not using them here, so they're just not needed. Using .53 will still work just fine in UWP apps using .NET 9, don't worry 🙂